android/app/src/main/kotlin/com/jabook/app/jabook/audio/AudioPlayerService.kt

```
@@ -523,6 +573,107 @@ public class AudioPlayerService : MediaLibraryService() {
+                        }
+
+                        LogUtils.i(
+                            "AudioPlayerService",
+                            "Book switch crossfade complete: ${filePaths.size} tracks, targetIndex=$targetIndex",
+                        )
+
+                        // Resume crossfade monitoring for chapter transitions
+                        crossfadeHandler?.startMonitoring()
+
+                        callback?.invoke(true, null)
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)